packages/flare/bin/cron_job_ingest_events.py

```diff
@@ -32,16 +33,33 @@ def main(logger: Logger, app: client.Application) -> None:
     if last_fetched_timestamp and last_fetched_timestamp > (
         datetime.now() - CRON_JOB_THRESHOLD_SINCE_LAST_FETCH
     ):
-        logger.debug(
+        logger.info(
             f"Fetched events less than {int(CRON_JOB_THRESHOLD_SINCE_LAST_FETCH.seconds / 60)} minutes ago, exiting"
         )
         return
 
     api_key, tenant_id = get_api_credentials(app=app)
 
-    fetch_and_print_feed_results(
+    save_last_fetched(app=app)
+    events_retrieved_count = 0
+    for event_feed in fetch_feed(
         logger=logger, app=app, api_key=api_key, tenant_id=tenant_id
-    )
+    ):
+        save_last_fetched(app=app)
+
+        # Rate limiting.
+        time.sleep(1)
+
+        save_start_date(app=app, tenant_id=tenant_id)
+        save_next(app=app, tenant_id=tenant_id, next=event_feed["next"])
+
+        if event_feed["items"]:
+            for item in event_feed["items"]:
+                print(json.dumps(item))
```

Comment:
I am wondering if flushing help would help with your "bug" of failing import during an exception.

You can `print(item, flush=True)`